source/_patterns/02-organisms/content-headers/content-header-profile.json

```diff
@@ -0,0 +1,19 @@
+{
+  "displayName": "Kananinoheaokuuhomeopuukaimanaalohilo",
+  "details": {
+    "affiliations": [
+      "University of Göttingen"
```

Comment:
Are we expecting department names here too (`Department of Underwater Basket Weaving, University of Nowheresville `), or only the top-level institution names?

Reply:
If we get institution data distinguishable from departmental data, then we should just use the institutional data.